packages/client/src/views/Home/queries.ts

```diff
@@ -81,3 +81,36 @@ export const FETCH_REGISTRATION_BY_COMPOSITION = gql`
     }
   }
 `
+
+export const FETCH_APPLICATION_SHORT_INFO = gql`
+  query data($id: ID!) {
+    fetchRegistration(id: $id) {
+      id
+      registration {
+        type
+        trackingId
+        status {
+          type
+        }
+      }
+      ... on BirthRegistration {
+        child {
+          name {
+            use
+            firstNames
+            familyName
+          }
+        }
+      }
+      ... on DeathRegistration {
+        deceased {
+          name {
+            use
+            firstNames
+            familyName
+          }
+        }
+      }
+    }
+  }
+`
```

Comment:
Change in plan, @Zangetsu101 has pointed out that the applications are being stored in the indexed DB. So lets use that instead of making a new GQL Query.